director/redirect.go

```diff
@@ -363,20 +363,20 @@ func ShortcutMiddleware(defaultResponse string) gin.HandlerFunc {
 func registerServeAd(ctx *gin.Context, sType ServerType) {
 	tokens, present := ctx.Request.Header["Authorization"]
 	if !present || len(tokens) == 0 {
-		ctx.JSON(401, gin.H{"error": "Bearer token not present in the 'Authorization' header"})
+		ctx.JSON(http.StatusUnauthorized, gin.H{"error": "Bearer token not present in the 'Authorization' header"})
```

Comment:
`401 -- unauthorized` is a bit of a misnomer in that it actually indicates unauthenticated (which is not something we check). The [HTTP specification RFC](https://www.rfc-editor.org/rfc/rfc9110.html#name-401-unauthorized) for `401` also states that if we want to return a `401`, we also need to start returning a `WWW-Authenticate` header with instructions for authentication. How about we just switch this to `http.StatusForbidden`.